src/pydata_sphinx_theme/assets/styles/index.scss

```
@@ -186,6 +188,11 @@ footer {
   &.no-sidebar {
     border-right: 0;
   }
+
+  .sidebar-end-items {
+    margin-top: auto;
+    margin-bottom: 0.5em;
```

```suggestion
    margin-bottom: 1em;
```

sounds good - we can always tweak it in the future too